### DISCUSSION TITLE
Does Tabby support bold fonts, so I can use POWERLEVEL9K_DIR_ANCHOR_BOLD in my prompt?

### DISCUSSION THREAD

**Discussion by pro-sumer** [Eugeny/tabby#6176] [Q&A] [answered]
My current terminal is iTerm and I there use the configuration option `POWERLEVEL9K_DIR_ANCHOR_BOLD` in [powerlevel10k](https://github.com/romkatv/powerlevel10k) to highlight significant parts of the path of the current working directory in bold font in my prompt.

I want those parts to be truly bold, not in a different color (tint).

Does Tabby (also) support this?

If so, how do I configure this?

**Comment by Eugeny** [comment, marked as answer]
Yes, Tabby supports bold fonts out of the box.

**Comment by pro-sumer** [comment]
> Tabby supports bold fonts out of the box

Oh, now I see...

I got distracted by the plain and bold text having different colours (because of a [temporary bug](https://github.com/Eugeny/tabby/discussions/6177)), but indeed not only the colours are different; the font is different too.

**Comment by lawweiliang** [comment]
@pro-sumer 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
